cluster/gce/gci/configure-helper.sh

```diff
@@ -751,6 +751,15 @@ function create-master-etcd-auth {
   fi
 }
 
+function create-master-encryption-provider {
```

Comment:
Can you move this to where we create the audit log config? I don't think it needs a top level function.